crosstool/cc_toolchain_config.bzl

```diff
@@ -675,8 +681,11 @@ def _impl(ctx):
         ctx.attr.cpu == "ios_arm64e" or
         ctx.attr.cpu == "ios_armv7" or
         ctx.attr.cpu == "ios_i386" or
-        ctx.attr.cpu == "ios_x86_64" or
         ctx.attr.cpu == "ios_sim_arm64" or
+        ctx.attr.cpu == "ios_x86_64" or
+        ctx.attr.cpu == "visionos_arm64" or
+        ctx.attr.cpu == "visionos_sim_arm64" or
+        ctx.attr.cpu == "visionos_x86_64" or
```

Comment:
This doesn't seem right? Shouldn't it have it's own for `-DOS_VISIONOS`? (Also, shouldn't watchOS have its own?)

Reply:
Looks like I took this from the original one https://github.com/bazelbuild/bazel/blob/c3cb69e11629455192650976a7026d51f7eedf3d/tools/osx/crosstool/cc_toolchain_config.bzl#L720-L736

maybe the intent was more "this platform is iOS like"

I think we should just not add visionOS here, I think folks should be using the platform provided ones of these instead anyways, pushed that